data/gui/widget/scroll_text_verbatim.cfg

```diff
@@ -0,0 +1,123 @@
+#textdomain wesnoth-lib
+
```

Comment:
Please add a comment about what this is, and hint about why it's separate from scroll_text_default.cfg.

Reply:
Updated.